test/setup.js

```diff
@@ -91,21 +90,81 @@ function screenshotCreateOrCompare (app, t, name) {
   })
 }
 
+// Compares two PNGs, ignoring any transparent regions in bufExpected.
+// Returns true if they match.
+function compareIgnoringTransparency (bufActual, bufExpected) {
```

Comment:
If we're doing this comparison while ignoring transparent regions, why is there an if-statement to not render the progress bar if `config.IS_TEST` is true in another file?